source/UI.cpp

```diff
@@ -270,6 +271,34 @@ Point UI::GetMouse()
 
 
 
+void UI::PlaySound(UISound sound)
+{
+	string name;
+	switch(sound)
+	{
+		case UISound::NORMAL:
+			name = "ui/click";
+			break;
+		case UISound::SOFT:
+			name = "ui/click_soft";
+			break;
+		case UISound::SOFT_BUZZ:
+			name = "ui/buzz_soft";
+			break;
+		case UISound::TARGET:
+			name = "ui/target";
+			break;
+		case UISound::FAILURE:
+			name = "fail";
```

Comment:
Maybe move this to `ui/`, too?

Reply:
I didn't want to break any plugins using or replacing that file.

Reply:
UI plugins tend to break the game (sometimes even making it temporarily unusable) if they're not updated regularly, so people shouldn't use outdated ones anyways.